mudlet-mapper.xml

```diff
@@ -10170,6 +10236,7 @@ function mmp_mapping_newroom(_, num)
               "."
           end
           -- check indoors status
+          if mmp.game ~= "asteria" then
```

Comment:
There seems to be some indentation missing here...

Reply:
Noted! Perhaps even mended.